client/src/app/site/motions/modules/motion-poll/motion-poll-detail/motion-poll-detail.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Location } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material';
+import { Title } from '@angular/platform-browser';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+
+import { MotionPollRepositoryService } from 'app/core/repositories/motions/motion-poll-repository.service';
+import { GroupRepositoryService } from 'app/core/repositories/users/group-repository.service';
+import { MotionPoll } from 'app/shared/models/motions/motion-poll';
+import { BaseViewComponent } from 'app/site/base/base-view';
+import { ViewMotionPoll } from 'app/site/motions/models/view-motion-poll';
+import { ViewGroup } from 'app/site/users/models/view-group';
+
+@Component({
+    selector: 'os-motion-poll-detail',
+    templateUrl: './motion-poll-detail.component.html',
+    styleUrls: ['./motion-poll-detail.component.scss']
+})
+export class MotionPollDetailComponent extends BaseViewComponent implements OnInit {
+    private pollId: number;
+
+    public pollTypes: object = { analog: 'Analog', named: 'Named', pseudonymous: 'Pseudonymous' };
+
+    public pollMethods: object = { YN: 'Yes/No', YNA: 'Yes/No/Abstain' };
+
+    public percentBase: object = {
```

Comment:
Two things:

1) The fields `majority_method` and `onehundred_percent_base` needs to be added to `base-poll.ts`. Please also add enums for these types.

2) In the `base-poll.ts` (as and special values in `motion-poll.ts` and `assignment-poll.ts`) there are enums describing, what values these special fields can have. Consider moving these objects into the classes to the enum. Please from the enum to the verbose names. Type: {[state: PollState]: string}` with PollState being the enum.